library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php

```diff
@@ -223,7 +223,7 @@ private function getActiveCalls(array $payload)
         return $response->result;
     }
 
-    public function isCgrEnabled()
+    public function isCgrEnabled(): bool|int
```

Comment:
Just a comment not meant to be handled in this pull request.

This returns non-boolean in case of error. I thing it would be more correct to raise an Exception in case of error instead of a multi-type return.